### DISCUSSION TITLE
How to add a link to an image...

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1003] [General]
There is no link field available for the image control. Using the link control I can only see a way to add a background image and that takes several steps to get margins and background size correct. Need to be able to simply drop an image on the page and wrap it with a url.

**Comment by meetmandeep** [comment]
You can drag any block within the Link Block. For example, you may want an image and a heading to link to a page. 

In your case, drag a Link Block to the page and drag an image within it.
